src/Microsoft.Azure.NotificationHubs/NotificationHubDescription.cs

```diff
@@ -0,0 +1,433 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Runtime.Serialization;
+using Microsoft.Azure.NotificationHubs.Messaging;
+using Microsoft.Azure.NotificationHubs.Common;
+
+namespace Microsoft.Azure.NotificationHubs
+{
+    /// <summary>
+    ///   Metadata description of the NotificationHub.
+    /// </summary>
+    [DataContract(Name = ManagementStrings.NotificationHubDescription, Namespace = ManagementStrings.Namespace)]
+    public sealed class NotificationHubDescription : EntityDescription
+    {
+        string path;
```

Comment:
Let's use the .NET Standard of using `_` prefix for member variables, which would then negate the need for `this.path = path` and instead would be `_path = path`